apps/browser-tests/src/main.ts

```diff
@@ -16,3 +19,63 @@ const unisatConnectButton = document.getElementById("unisat-connect");
 if (unisatConnectButton) {
   unisatConnectButton.addEventListener("click", connectToUnisat);
 }
+
+async function createAndPreparePsbt() {
+  const psbtParams = {
+    address: "tb1p98dv6f5jp5qr4z2dtaljvwrhq34xrr8zuaqgv4ajf36vg2mmsruqt5m3lv",
+    feeRate: 1,
+    publicKey:
+      "039ce27aa7666731648421004ba943b90b8273e23a175d9c58e3ec2e643a9b01d1",
+    outputs: [
+      {
+        address: "tb1qatkgzm0hsk83ysqja5nq8ecdmtwl73zwurawww",
+        value: 600,
+      },
+    ],
+    network: "testnet" as Network,
+  };
+  const psbt = new PSBTBuilder(psbtParams);
+  console.log("Initial Psbt: ", psbt);
+
+  // Duplicate the PSBT so that console log shows a different instance.
+  const clonedPSBT = new PSBTBuilder(psbtParams);
+  await clonedPSBT.prepare();
+  console.log("Prepared Psbt: ", clonedPSBT);
+  return clonedPSBT;
+}
+
+async function handleCreateAndPreparePsbtButtonClick() {
+  await createAndPreparePsbt();
+}
+
+const createPsbtButton = document.getElementById("create-psbt");
+if (createPsbtButton) {
+  createPsbtButton.addEventListener(
```

Comment:
After this, let's move this browser-tests to at least a proper lib. :D

Reply:
Added issue #62